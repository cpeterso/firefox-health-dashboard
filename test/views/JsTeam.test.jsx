import React from 'react';
import renderer from 'react-test-renderer';
import JsTeam from '../../src/views/JsTeam';

it('renders correctly', () => {
  const tree = renderer.create(<JsTeam />).toJSON();

  expect(tree).toMatchSnapshot();
});
