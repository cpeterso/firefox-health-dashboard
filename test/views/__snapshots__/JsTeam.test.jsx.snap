// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="DashboardPage-root-2"
>
  <div
    className="DashboardPage-title-3"
  >
    <h1
      style={
        Object {
          "display": "inline",
        }
      }
    >
      JavaScript Performance dashboard
      <small
        className="DashboardPage-subtitle-4"
      />
    </h1>
    <span
      style={
        Object {
          "float": "right",
        }
      }
    >
      <a
        href="https://github.com/mozilla-frontend-infra/firefox-health-dashboard/"
      >
        <img
          alt="Go to Github Repo"
          src="test-file-stub"
          style={
            Object {
              "padding": "0 0.5rem 0 0",
              "width": "2rem",
            }
          }
        />
      </a>
    </span>
  </div>
  <div
    className="JsTeam-content-1"
  >
    Under construction. Welcome to the future home of arewefastyet.com!
  </div>
</div>
`;
